WORKSPACE

```
@@ -257,6 +257,10 @@ maven_install(
         # Any two platforms to ensure that it doesn't work _only_ under the host operating system
         "com.google.protobuf:protoc:exe:linux-x86_64:3.21.12",
         "com.google.protobuf:protoc:exe:osx-aarch_64:3.21.12",
+        # https://github.com/bazelbuild/rules_jvm_external/issues/917
+        # androidx core-testing POM has "exclusion" for "byte-buddy" but it should be downloaded as mockito-core dependency.
+        "org.mockito:mockito-core:jar:3.3.3",
+        "androidx.arch.core:core-testing:aar:2.1.0",
```

I gave a try with these 2 artifacts exactly as written above and indeed faced the issue that `net.bytebuddy:byte-buddy` and `net.bytebuddy:byte-buddy-agent` weren't fetched.
But, despite the theoretical equivalence, stripping out the `jar` packaging type seems to lead to the intended behavior.
```suggestion
        "org.mockito:mockito-core:3.3.3",
        "androidx.arch.core:core-testing:aar:2.1.0",
```
Looking...

---

@artem-zinnatullin I just filed https://github.com/artem-zinnatullin/rules_jvm_external/pull/1 you may merge to your branch or not, at your convenience.

---

Merged!